workflow/rules/common.smk

```diff
@@ -12,6 +12,11 @@ from snakemake.utils import validate
 
 VARTYPES = ["SNV", "MNV", "INS", "DEL", "REP", "INV", "DUP"]
 
+# clear text / content of flag "technology" in sample sheet
+ILLUMINA = "illumina"
+ONT = "ont"
+
```

Comment:
I'm really a little worried how this thing grew now. Many kind of similar names and loads of functions. Heavily
Overcrowded

Reply:
Totally agree. We have to refactor this soon. Maybe we can do this while addressing #357 ?